roles/sap_anydb_install_oracle/.yamllint.yml

```diff
@@ -8,7 +8,9 @@ rules:
   brackets: {max-spaces-inside: 1, level: error}
 #  colons: {max-spaces-after: -1, level: error}
 #  commas: {max-spaces-after: -1, level: error}
-  comments: disable
+  comments:
+    require-starting-space: false
```

Comment:
The min-spaces-from-content is good, but why the `require-starting-space`? Reading the documentation:

```
Use require-starting-space to require a space character right after the #. Set to true to enable, false to disable.
```

Even stick with the default value `true` is ok in our case, or not?

Reply:
As per the output I have shown in https://github.com/sap-linuxlab/community.sap_install/pull/811, ansible-lint 24.7.0 reported `incompatible custom yamllint configuration` and demanded `comments.min-spaces-from-content must be 1` if a custom yamllint configuration is desired (which we want/need).

When *not* also setting `require-starting-space: false` in the `comments:` section, ansible-lint will report many errors of type:
```
yaml[comments]: Too few spaces before comment
```

for each role.